src/Cms/AbstractWebTemplate.php

```diff
@@ -289,4 +360,93 @@ public function twitterCardImage()
 
         return $this->metaImage();
     }
+
+
+
+    // Polylingual
+    // =================================================================================================================
+
+    /**
+     * Set the available languages.
+     *
+     * @param  array $languages The list of languages.
+     * @return self
+     */
+    protected function setAvailableLanguages(array $languages)
+    {
+        $this->availableLanguages = $languages;
+
+        return $this;
+    }
+
+    /**
+     * Retrieve the translator service.
+     *
+     * @return array
+     */
+    protected function availableLanguages()
+    {
+        return $this->availableLanguages;
+    }
+
+    /**
+     * Retrieve the locale shortcode
+     *
+     * @return string
+     */
+    public function currentLanguage()
+    {
+        return $this->translator()->getLocale();
+    }
+
+    /**
+     * Determine if there exists alternate translations associated with the current route.
+     *
+     * @return boolean
+     */
+    public function hasAlternateTranslations()
+    {
+        return count(iterator_to_array($this->alternateTranslations())) > 0;
+    }
+
+    /**
+     * Render the alternate translations associated with the current route.
+     *
+     * This method _excludes_ the current route's canonical URI.
+     *
+     * @return Generator|null
+     */
+    public function alternateTranslations()
+    {
+        if ($this->alternateTranslations === null) {
+            $context = $this->contextObject();
+            $isModel = ($context instanceof ModelInterface);
+            $isRoutable = ($context instanceof RoutableInterface);
+
+            $origLang   = $this->currentLanguage();
+
+            foreach ($this->availableLanguages() as $lang) {
+                if ($lang === $origLang) {
+                    continue;
+                }
+
+                $this->localesManager->setCurrentLocale($lang);
+
+                $data = [
```

Comment:
Let's move the array format to a separate method for easy customization:
`protected function formatAlternateTranslation()`